src/wscat.ts

```diff
@@ -58,7 +60,7 @@ function setup(options: IOptions, socket: WebSocket) {
 }
 
 export function connect(options: IConnectOptions) {
-    const socket = new WebSocket(options.address, options)
+    const socket = new WebSocket(options.address, undefined, options)
```

Comment:
Why the undefined here?

Reply:
Because tslint on my machine complained without it.
```
  /**
   * Create a new `WebSocket`.
   *
   * @param {(String|url.Url|url.URL)} address The URL to which to connect
   * @param {(String|String[])} protocols The subprotocols
   * @param {Object} options Connection options
   */
```
For some reason tslint is happy to accept the 2-arg form as it is on master currently, but apparently the addition of a string[] member to the options object makes tslint less tolerant.

The undefined makes this invocation match the constructor’s documented signature, which silences the tslint warnings.

This having been said, I’ve done a lot of JS but almost zero TS; maybe there’s a better approach? Might be better as an array reference?

Reply:
Ah, might just be the 3rd party typedef that's out of sync. I'll try updating it when I tag a release and see if the error goes away.

Reply:
BTW, thank you for maintaining this project. I’m surprised that the “upstream” project doesn’t offer a binary mode.

Reply:
Published as 2.1.0

Yeah upstream is more of a `term` than a `cat` in my opinion. Hard to support a binary option when you can't pipe it somewhere :)